DataProcessor/src/net/relinc/processor/controllers/TrimDataController.java

```diff
@@ -757,155 +752,87 @@ private int getChartBeginIndex(){
 	
 	private int getChartEndIndex(){
 		if(xAxis.isAutoRanging()){
-        	return getActivatedData().Data.timeData.length - 1;
+        	return getActivatedData().getModifiedTime().length - 1;
         }
 		return getActivatedData().getIndexFromTimeValue(xAxis.getUpperBound() / timeUnits.getMultiplier());
 	}
 
 	private void updateChart() {
 		if(listView.getSelectionModel().getSelectedIndex() == -1)
 			return;
-		double[] xData = getActivatedData().Data.timeData;
-		double[] yData = getActivatedData().Data.data;
-		double[] filteredYData = yData.clone();
-		double[] pochammerAdjustedData = new double[getActivatedData().getEnd() - getActivatedData().getBegin() + 1];
-		double[] zeroedData = yData.clone();
-		double[] fittedData = yData.clone();
-		
-		if(getActivatedData().modifiers.getLowPassModifier().activated.get()){
-			filteredYData = SPMath.fourierLowPassFilter(filteredYData, getActivatedData().modifiers.getLowPassModifier().getLowPassValue(), 1.0 / (xData[1] - xData[0]));
-		}
-		if(getActivatedData().modifiers.getZeroModifier().activated.get()){
-			zeroedData = SPMath.subtractFrom(zeroedData, ((ZeroOffset)getActivatedData().modifiers.getModifier(ModifierEnum.ZERO)).getZero());
-		}
-		
-		if(getActivatedData().modifiers.getPochammerModifier().activated.get()){
-			if(getActivatedData() instanceof HopkinsonBarPulse){
-				HopkinsonBarPulse pulse = (HopkinsonBarPulse)getActivatedData();
-				pochammerAdjustedData = pulse.getPochammerAdjustedArray(barSetup);
-			}
-		}
-		
-		if(getActivatedData().modifiers.getFitterModifier().activated.get()){
-			Fitter fitter = getActivatedData().modifiers.getFitterModifier();
-			fittedData = fitter.applyModifierToData(getActivatedData().Data.data.clone(), getActivatedData());
+		if(getActivatedData().getModifiers().getResamplerModifier().activated.get()) {
+			System.out.println();
 		}
-		
+		double[] xData = getActivatedData().getModifiedTime();
+		double[] yData = getActivatedData().getModifiedData();
+
+
 		XYChart.Series<Number, Number> rawDataSeries = new XYChart.Series<Number, Number>();
-		XYChart.Series<Number, Number> filteredDataSeries = new XYChart.Series<Number, Number>();
-		XYChart.Series<Number, Number> pochammerSeries = new XYChart.Series<Number, Number>();
-		XYChart.Series<Number, Number> zeroedSeries = new XYChart.Series<Number, Number>();
-		XYChart.Series<Number, Number> fittedSeries = new XYChart.Series<Number, Number>();
```

Comment:
it really didn't make sense to graph each modified dataset separately, because in the viewer, all the modifiers are run. So I've made it so it matches the viewer, and graphs the one modified dataset.